src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,46 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  .govuk-\!-font-weight-regular {
+    font-weight: normal !important;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    .govuk-\!-summary-list-key-width {
+      width: 50% !important;
+    }
+
+    .govuk-\!-summary-list-actions-width {
+      width: 50% !important;
+    }
+  }
```

Comment:
These already exist in the gov.uk design system so we can get rid of these completely and just use them directly in `src/components/list-with-actions/template.njk`